cosmwasm/packages/std/src/query.rs

```diff
@@ -144,6 +159,24 @@ pub enum StakingQuery {
     UnbondingDelegations { delegator: HumanAddr },
 }
 
+/// ProposalsResponse is data format returned from GovQuery::Proposals query
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct ProposalsResponse {
+    pub proposals: Vec<Proposal>,
+}
+
+/// ProposalsResponse is data format returned from GovQuery::Proposals query
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct Proposal {
+    pub id: u64,
+    /// Time of the block where MinDeposit was reached. -1 if MinDeposit is not reached
+    pub voting_start_time: u64,
+    /// Time that the VotingPeriod for this proposal will end and votes will be tallied
+    pub voting_end_time: u64,
```

Comment:
This never happens. The comment is just copy pasted from somewhere else

Reply:
Then it's best to change it, to prevent confusion.
If this condition is true, then it would just not list it in the response, so the contract would not observe it?